_data/canonical-tags/community.yaml

```diff
@@ -0,0 +1,3 @@
+id: community
+name: Community
+description: Related to Kubernetes open-source development.
```

Comment:
I was originally going to change the hell out of this, but the more I think about it, the more I think this works.